Header header
GroundTargetState[] ground_targets